requirements.in

```diff
@@ -26,6 +26,7 @@ pytz
 redis==3.2.0
 requests>=2.20
 sentry-sdk==0.14.3
+six==1.15.0
```

Comment:
If this is required for testing we should pin it in `test_requirements.txt` instead

Reply:
Updated